posthog/hogql_queries/insights/funnels/base.py

```diff
@@ -898,7 +902,12 @@ def _get_breakdown_prop_expr(self, group_remaining=False) -> List[ast.Expr]:
                 BreakdownType.group,
             ]:
                 breakdown_values = self._get_breakdown_conditions()
-                return [parse_expr(f"if(has({breakdown_values}, prop), prop, {other_aggregation}) as prop")]
+                return [
+                    parse_expr(
+                        f"if(has({{breakdown_values}}, prop), prop, {other_aggregation}) as prop",
```

Comment:
The more things we can move to placeholders overall, the better. Could `other_aggregation` also go? It's equally a constant. Same with possibly `self._get_breakdown_prop()` above?

Reply:
Jup, I agree. Would be great to even have a lint rule warning about f-strings with `parse_expr`. 

For now I went over all usages of `parse_expr` in funnels and checked that the interpolated strings work as expected e.g. there are a lot of places where some number is interpolated `latest_{i}` and `i` is validated via pydantic.